server/lib/src/idm/group.rs

```diff
@@ -280,99 +280,44 @@ pub(crate) fn load_all_groups_from_account<'a, E, TXN>(
 ) -> Result<(Vec<Group<()>>, Vec<Group<Unix>>), OperationError>
 where
     E: Committed,
+    Entry<E, EntryCommitted>: GetUuid,
     TXN: QueryServerTransaction<'a>,
 {
     let Some(iter) = value.get_ava_as_refuuid(Attribute::MemberOf) else {
         return Ok((vec![], vec![]));
     };
 
-    let f = filter!(f_or(
-        iter.map(|u| f_eq(Attribute::Uuid, PartialValue::Uuid(u)))
-            .collect()
-    ));
+    let conditions: Vec<_> = iter
+        .map(|u| f_eq(Attribute::Uuid, PartialValue::Uuid(u)))
+        .collect();
+
+    println!("{:?}", conditions);
+
+    let f = filter!(f_or(conditions));
 
     let entries = qs.internal_search(f).map_err(|e| {
         admin_error!(?e, "internal search failed");
         e
     })?;
 
-    println!("entries: {:?}", entries);
+    println!("{}", entries.len());
 
-    let mut unix_groups = vec![];
+    // Do we need the user group for non unix?
```

Comment:
For non-unix, no.

Reply:
Easy. Had a feeling. I'll remove that comment and then it's ready to go

Reply:
Still reading :)